### DISCUSSION TITLE
Decide on build system (Gradle vs Maven)

### DISCUSSION THREAD

**Discussion by slachiewicz** [ChargeTimeEU/Java-OCA-OCPP#316] [Polls]
Would be good to stick to one build system because now if we made improvements (code formatting plugins, dependency updates) we need to change in both configurations. 
For Maven we have set up to push artifacts, we use it in GitHub Actions verification, but with Gradle, we do code formatting with pre-commit check.

**Comment by TVolden** [comment]
I have moved this to the discussion section. Hopefully, the community will chip in with their preference and arguments.
The polls doesn't seem to work in this one, so I'm creating a new thread for the vote
